duckdb-vortex/extension_config.cmake

```diff
@@ -5,3 +5,6 @@ duckdb_extension_load(vortex
     SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}
     LOAD_TESTS
 )
+
+duckdb_extension_load(tpch)
```

Comment:
?

Reply:
We need this to generate tpch data, can remove if we load the ext into regular duckdb I uess

Reply:
I think we should add a comment here that these extensions only get loaded automatically when they get linked statically. And that this is not the case when the Vortex extension gets dynamically loaded, e.g. as a community extension.